CMakeLists.txt

```diff
@@ -1078,6 +1078,10 @@ zephyr_compile_options(
   $<$<COMPILE_LANGUAGE:C>:-std=${CSTD}>
 )
 
+configure_file(
+     $ENV{ZEPHYR_BASE}/include/arch/arm/cortex_m/scripts/linker_section_alignment.ld
```

Comment:
${ZEPHYR_BASE}

Reply:
Yeap, I can update this. (copy-pasted from following configure_file( statements)